### ISSUE TITLE
Authentication

### ISSUE THREAD

**Issue by joeyjoejoejr** [aellispierce/bull_city_vegan_challenge#3]
We should set up some kind of authentication.  The standard sign up flow , but it might also be nice to throw a facebook OAuth signup flow and/or other third-party applications.

**Comment by aellispierce** [comment]
Yep, I was thinking facebook would be ideal for this kind of application. That way we could also possibly add a "share" feature so that they could let others know they'd participated in the challenge and help generate buzz
